src/components/Time.tsx

```diff
@@ -0,0 +1,32 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+
+import React, { useState, useEffect } from "react";
+
+export default function Time() {
+  const [currentTime, setCurrentTime] = useState(new Date());
+
+  const hours = currentTime.getHours();
+  const minutes = currentTime.getMinutes();
+  const seconds = currentTime.getSeconds();
+
+  useEffect(() => {
+    setInterval(() => {
+      setCurrentTime(new Date());
+    }, 0);
```

Comment:
```suggestion
    }, 1000);
```

이렇게 작성해주셔야해요!

매초마다 스테이트를 갱신하면 되기 때문이에요.

지금처럼 0을 적어주신다면 매 ms(밀리세컨드)마다 state를 갱신하기 때문에 필요하지 않은 갱신이 일어난다고 볼 수 있죠!